components/layout/positioned.rs

```diff
@@ -774,6 +776,18 @@ impl AbsoluteAxisSolver<'_> {
         )
     }
 
+    #[inline]
+    fn solve_size_lazily(&self) -> LazySize<'_> {
+        LazySize::new(
+            &self.computed_sizes,
+            self.axis,
+            self.automatic_size(),
+            Au::zero,
+            Some(self.stretch_size()),
+            self.is_table,
+        )
+    }
```

Comment:
Will this be used in more than one place? If not, I would just remove the function and write it inline.

Reply:
OK, with this PR `solve_size_lazily`, `solve_size_extrinsically` and `solve_size` have each a single caller, so I will inline them all.